images/win/scripts/ImageHelpers/InstallHelpers.ps1

```diff
@@ -161,6 +161,75 @@ Hashtable for service arguments
     }
 }
 
+function Install-VsixExtension
+{
+    Param
+    (
+        [String]$Url,
```

Comment:
nitpick: uri instead of string

Reply:
Let's leave Url for consistency – all the other helpers function use Url, not Uri

Reply:
Actually, I'm talking about argument type. [uri] instead of [string]